### PR TITLE
Enable filepath and time BBEs

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

File: examples/filepath/tests/filepath_test.bal [hunk 1]
```diff
@@ -17,7 +16,7 @@ public function mockPrint(any|error... s) {
 
 boolean isWindows = system:getEnv("OS") != "";
 
-@test:Config
+@test:Config {}
```
Comment (niveathika) [inline examples/filepath/tests/filepath_test.bal:19]: Shall we remove this test? Since the output is verified in the build?

Comment (ThishaniLucas) [reply examples/filepath/tests/filepath_test.bal]: done

**Review by niveathika — APPROVED** [review]

**Comment by daneshk** [comment]
@ThishaniLucas why are we removing the test case?

**Comment by ThishaniLucas** [comment]
> @ThishaniLucas why are we removing the test case?

Because both the testcase and the output verifaction are doing the same thing. Therefore, it seems we can remove either.

**Comment by daneshk** [comment]
> > @ThishaniLucas why are we removing the test case?
> 
> Because both the testcase and the output verifaction are doing the same thing. Therefore, it seems we can remove either.

For output verification, we need to run it manually right?. if we have the test case, it will run and verify each build

**Comment by niveathika** [comment]
@daneshk No the output verification task is added in the build itself unless there are ignored explicitly all the BBEs output will be verified against .out files.

**Comment by daneshk** [comment]
> @daneshk No the output verification task is added in the build itself unless there are ignored explicitly all the BBEs output will be verified against .out files.

Got it. then it's fine

**Pull request merged**